lib/braid/command.rb

```diff
@@ -114,21 +114,21 @@ def determine_repository_revision(mirror)
       if mirror.tag
         if use_local_cache?
           Dir.chdir git_cache.path(mirror.url) do
-            git.rev_parse(mirror.local_ref)
+            git.rev_list_1(mirror.local_ref)
```

Comment:
`git.rev_parse(mirror.local_ref + '^{commit}')` would be more explicit, here and in the other places.  Please add a comment about why this is necessary.

Reply:
thanks for the hint, i changed this